libs/community/langchain_community/embeddings/gradient_ai.py

```diff
@@ -70,23 +67,26 @@ def validate_environment(cls, values: Dict) -> Dict:
         )
 
         values["gradient_api_url"] = get_from_dict_or_env(
-            values, "gradient_api_url", "GRADIENT_API_URL"
+            values, "gradient_api_url", "GRADIENT_API_URL", default=None
         )
+        try:
+            import gradientai
+        except ImportError:
+            raise ImportError(
+                "GradientEmbeddings requires `pip install gradientai>=1.4.0`."
```

Comment:
should we make this `pip install -U gradientai`? seems more future proof?

Reply:
Thanks, resolved that. 

Is there a way to make a `optional` extra in pyproject.toml of langchain community?
Aka, `pip install langchain-community[gradientai]`

This would warn the user much earlier than raising importerror if the package version is too low.